CONTRIBUTING.md

```diff
@@ -63,4 +63,4 @@ COMING SOON
 By contributing, you agree that your contributions will be licensed under its [AGPL](LICENSE) License.
 
 ## References
-This document was adapted from the open-source contribution guidelines for [Facebook's Draft](https://github.com/facebook/draft-js/blob/main/CONTRIBUTING.md)
\ No newline at end of file
+This document was adapted from the open-source contribution guidelines for [Facebook's Draft](https://github.com/facebook/draft-js/blob/main/CONTRIBUTING.md)
```

Comment:
I'm not sure why github thinks this line was changed, as far as I can tell no change was made, and it's substituting an identical text at the same line

Reply:
that's weird... Maybe some sneaky whitespace or something creeping around in there?